packages/macro/src/macroJs.ts

```diff
@@ -467,8 +562,10 @@ export default class MacroJs {
     return (
       this.types.isTaggedTemplateExpression(node) &&
       (this.isLinguiIdentifier(node.tag, "t") ||
+        this.isLinguiIdentifier(node.tag, "_") ||
         (this.types.isCallExpression(node.tag) &&
-          this.isLinguiIdentifier(node.tag.callee, "t")))
+          (this.isLinguiIdentifier(node.tag.callee, "t") ||
+            this.isLinguiIdentifier(node.tag.callee, "_"))))
```

Comment:
This solution is temporary here to make recursive macro matching work.  What is really needed is extend `isRootPath` to understand calls of function returned by `useLingui` 

Other option would be use babel-macro only as "backdoor" which will allow running arbitrary transformation code and then write whole functionality as regular babel plugin with visitor pattern. 

A little bit of context here: 

babel's visitor traverse the AST in depth first algorithm. That completely eliminates a problem with nested macro calls such as: 

```js
t`Hello ${plural(...)}`
```

Because we can process them from root and expand recursively. 

However, with babel macro plugin we receive list of "references" which is flat and in undefined order. 
For example above we will receive 2 references `t` and `plural` and we should process only `t` and ignore `plural` since it's a part of outer `t` expression.